servicemesh-install/topics/tutorial-kiali.adoc

```diff
@@ -37,61 +37,101 @@ tracing                tracing-istio-system.127.0.0.1.nip.io
+
+. Click *Graph* in the left navigation.
 +
-image:kiali-graph.png[kiali graph]
+image:kiali-graph.png[Kiali graph]
 +
-. From the *Namespace* menu, select `bookinfo`.  Now the graph displays only the services in the Bookinfo application.
-. Click *Legend* in the lower left corner.  Kiali displays a window that contains the graph legend.
+. If necessary, select `bookinfo` from the *Namespace* menu.  The graph displays the applications in the Bookinfo application.
+. Click the question mark (?) under the *Namespace* menu to take the Graph Help Tour.
+. Click *Done* to close the Help Tour.
+. Click *Legend* in the lower left corner.  Kiali displays the graph legend.
 +
-image:kiali-legend.png[kiali legend]
+image:kiali-legend.png[Kiali legend]
 +
 . Close the Graph Legend.
 . Hover over the *productpage* node.  Note how the graph highlights only the incoming and outgoing traffic from the node.
-. Click the *productpage* node.  Note how the details on the right side of the page change to show the *productpage* details.
+. Click the *productpage* node.  Note how the details on the right side of the page change to display the *productpage* details.
+
+[[kiali-applications-page]]
+== Applications page
+The Applications page lets you search for and view applications, their health, and other details.
+
+. Click *Applications* in the left navigation.
+. If necessary, select `bookinfo` from the *Namespace* menu.  The page displays the applications in the selected namespace and their health.
+. Hover over the Health icon to view additional health details.
+. Click the `reviews` service to view the details for that application.
++
+image:kiali-applications-details.png[Kiali Applications details]
++
+. On the Applications Details page you can view more detailed health information, and drill down for further details about the three versions of the `reviews` service.
+. From the Application Details page you can also click tabs to view Traffic and Inbound and Outbound Metrics for the application.
+
+[[kiali-workloads-page]]
+== Workloads page
+The Workloads page lets you search for and view workloads, their health, and other details.
+
+. Click *Workloads* in the left navigation.
+. If necessary, select `bookinfo` from the *Namespace* menu.  The page displays the workloads in the selected namespace, their health, and labels.
+. Click the `reviews-v1` workload to view the details for that workload.
+. On the Workload Details page you can view an overview of pods and services associated with the workload.
++
+image:kiali-workloads-details.png[Kiali Workloads details]
++
+. From the Workload Details page you can also click tabs to view Traffic, Logs, and Inbound and Outbound Metrics for the workload.
 
 [[kiali-services-page]]
 == Services page
+The Services page lets you search for and view services, their health, and other details.
```

Comment:
It happens the same with the service details page.
"other details" sounds generic when probably we can describe the options user can see from these page.